css/css-text-decor/parsing/text-decoration-valid.html

```diff
@@ -2,7 +2,8 @@
 <meta charset="utf-8">
 <title>CSS Text Decoration Test: Parsing text-decoration with valid values</title>
 <link rel="help" href="https://drafts.csswg.org/css-text-decor-3/#propdef-text-decoration">
-<meta name="assert" content="text-decoration supports the full grammar '<‘text-decoration-line’> || <‘text-decoration-style’> || <‘text-decoration-color’>'.">
+<link rel="help" href="https://drafts.csswg.org/css-text-decor-4/#text-decoration-width-property">
+<meta name="assert" content="text-decoration supports the full grammar '<‘text-decoration-line’> || <‘text-decoration-style’> || <‘text-decoration-color’> || <'text-decoration-thickness>'.">
```

Comment:
text-decoration-width is not yet tested in the file.

Thus we should not yet mention css-text-decor-4 or text-decoration-width.

Reply:
I see. I will drop this.

Reply:
Actually we did. At least these three tests are related to text-decoration-thickness (which has been renamed from text-decoration-width). However, the property name is text-decoration, so ok I will drop this. Thanks. 
```
test_valid_value("text-decoration", "auto", "none");
test_valid_value("text-decoration", "from-font");
test_valid_value("text-decoration", "10px");
```